godeltaprof/README.md

```diff
@@ -1 +1,98 @@
 # godeltaprof
+
+godeltaprof is an efficient delta profiler for memory, mutex, block.
+
+# Why
+
+In golang, allocation, mutex and block profiles are cumulative - they only grow over time and show allocations happened since the beginning of the running program.
```

Comment:
When used in other docs (for example [Golang tutorial](https://www.golangprograms.com/go-language.html)), it looks like Golang is usually capitalized. Is that how you usually use it? 
```suggestion
In Golang, allocation, mutex and block profiles are cumulative. They only grow over time and show allocations that happened since the beginning of the running program.
```